EnvChanger/Classes/EnvChangerController.swift

```diff
@@ -133,7 +195,8 @@ public class EnvChangerController<T>: UIViewController where T: EnvironmentRepre
         
         button.sizeToFit()
         /// Starting point of the button is always in the top left corner.
-        button.frame = CGRect(origin: CGPoint(x: view.frame.minX + 60, y: view.frame.minY + 60), size: button.bounds.size)
+        button.frame = CGRect(origin: CGPoint(x: view.frame.minX + 60, y: view.frame.minY + 60),
```

Comment:
[optional]

The applied value `60` can be moved to a constant.

Reply:
Done.